### PR TITLE
Do not call GetTerminalSize if STDOUT is not a tty

### PR DESCRIPTION
**Pull request by mscherer**
Followup of the bug fix in #86, triggered by running needrestart
with yum/dnf in cron

### CODE REVIEW

File: perl/lib/NeedRestart/UI.pm [hunk 1]
```diff
@@ -129,8 +129,9 @@ sub _progress_inc {
 
 sub _progress_out {
     my $self = shift;
-    
-    my ($columns) = GetTerminalSize(\*STDOUT);
+    my $columns = 10;
```
Comment (liske) [inline perl/lib/NeedRestart/UI.pm]: Please use 80 as fallback value.

File: perl/lib/NeedRestart/UI.pm [hunk 2]
```diff
@@ -142,10 +143,11 @@ sub _progress_out {
 
 sub _progress_fin {
    my $self = shift;
+   my $columns;
```
Comment (liske) [inline perl/lib/NeedRestart/UI.pm]: The variable should also be set to 80 to prevent perl warnings about using undefined value in line 152.

**Comment by mscherer** [comment]
Done

**Comment by liske** [comment]
Thanks!

**Pull request merged**